etc/kubernetes/operator-debezium/lgarciaac/app-config/application.properties

```diff
@@ -9,4 +9,5 @@ quarkus.log.category."io.fabric8.kubernetes.client.internal.VersionUsageUtils".l
 # operator
 #
 cos.connectors.namespace = ${kubernetes.namespace}
-cos.operators.namespace   = ${kubernetes.namespace}
\ No newline at end of file
+cos.operators.namespace   = ${kubernetes.namespace}
+cos.operator.debezium.image-pull-secrets-name = "addon-pullsecret"
```

Comment:
looks like `addon-pullsecret` is already the default value, not sure if we need tho set the value in the property files

Reply:
Right, I will remove it from the property files.